### PR TITLE
[LW-01] Create wallet sql store

### PR DESCRIPTION
**Pull request by jesusx21**
Starting the project with python, the first step to take on is create wallet store, we start with the sql store, tests are pending

image
image

### CODE REVIEW

**Comment by jesmoo** [comment]
like, o como doy like aqui?xd

File: database/stores/errors.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class DatabaseError(Exception): pass # noqa
+class NotFound(DatabaseError): pass # noqa
+
+
+class InvalidId(DatabaseError):
+    def __init__(self, id):
+        self.id = id
```
Comment (IngMoisesToro) [inline database/stores/errors.py:7]: Este es el constructor?

**Comment by IngMoisesToro** [comment]
Like

**Review by IngMoisesToro — APPROVED** [review]

**Review by jesmoo — APPROVED** [review]

File: database/stores/errors.py (hunk 1 above)
Comment (jesusx21) [reply database/stores/errors.py:7]: correcto

**Pull request merged**